app/controllers/concerns/user_settable.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
why it need? remove it from the places where it does not require.

Reply:
@kth-wtag apu, I think what it does is to make each string literal declared in the file immutable which in Ruby are mutable by default otherwise. Also, not doing so results in warning.

Reply:
> @kth-wtag apu, I think what it does is to make each string literal declared in the file immutable which in Ruby are mutable by default otherwise. Also, not doing so results in warning.

what warning it gives?

Reply:
![image](https://github.com/wsa-wtag/medical-appointment-scheduler/assets/167951273/a35afdee-9f7f-4f57-b41b-90e86bb6480f)
@kth-wtag apu, this warning.